Src/PCompiler/PCommandLine/Options/PCheckerOptions.cs

```diff
@@ -238,5 +248,41 @@ private static void SanitizeConfiguration(CheckerConfiguration checkerConfigurat
                 Error.ReportAndExit("For the option '-max-steps N[,M]', please make sure that M >= N.");
             }
         }
+        
+
+        private static void FindLocalPCompiledFile(CheckerConfiguration checkerConfiguration)
+        {
+            if (checkerConfiguration.AssemblyToBeAnalyzed == string.Empty)
+            {
+                CommandLineOutput.WriteInfo(".. Searching for a P compiled file locally in the current folder");
+                
+                string filePattern =  checkerConfiguration.CheckerMode switch
+                {
+                    "bugfinding" => "*.dll",
+                    "verify" => "*-jar-with-dependencies.jar",
+                    "cover" => "*-jar-with-dependencies.jar",
+                    "pobserve" => "*-jar-with-dependencies.jar",
+                    _ => "*.dll"
+                };
+                
+                var files = Directory.GetFiles(Directory.GetCurrentDirectory(), filePattern, SearchOption.AllDirectories);
+
+                foreach (var fileName in files)
+                {
+                    if (fileName.EndsWith("PCheckerCore.dll") || fileName.EndsWith("PCSharpRuntime.dll")) continue;
```

Comment:
How about searching for a file with the project name? Which is what we use as the generated dll name?

Reply:
`p check` command does not know the project name as it doesn't have the option `--projname` right now.
Do you suggest adding such an option?

Reply:
Can you clarify what you mean by "unused field error"?